### PR TITLE
New landing and galaxy page 

### LINKED ISSUE #99: Landing page UI changes
## Design changes

image

image

## Tasks

- [x] Design new bg images @mosaeedi 
- [ ] Replace bg images in code
- [x] Update text content
- [ ] Replace header logo with new pairwise logo

### PR DESCRIPTION
**Pull request by yussdev**
Closes #99 #100 #102

### CODE REVIEW

**Comment by kristoferlund** [comment]
First look, will look more in 20 mins
* Apply `git merge staging` and resolve any confilicts
* Last section have some gfx glitches:
  - Format numbers US style `100,234.99`
  - Replace squares with locks

File: src/pages/index.tsx [hunk 1]
```diff
@@ -94,49 +97,50 @@ export default function Home() {
...
         </div>
         <VectorHorizontal3 className="mx-auto" />
         <div className="my-6 -ml-36 flex w-[494px] flex-col gap-6 rounded-[32px] border border-white px-11 pb-14 pt-24 text-center text-black">
-          <h3 className="text-[32px] font-bold">Unlock More Collections</h3>
+          <h3 className="text-[32px] font-bold">Leveraging Your Expertise</h3>
           <p className="text-lg font-medium">
-            Unleash your influence by voting on the unlocked collection. Your
-            votes are your voice, shaping the future of these projects. Every
-            click is a step toward making a difference.
+            Pairwise streamlines the review process, guiding you to the
```
Comment (kristoferlund) [inline src/pages/index.tsx:127]: @mosaeedi 

Graphics still uses wrong terms. `Begin ranking` should be used, not `Awaiting direction`.

image

**Review by kristoferlund — CHANGES_REQUESTED** [review]
I'm unable to continue review until you merge branch with staging @yussdev. Now it is using api calls that don't exist anymore.

File: src/pages/index.tsx (hunk 1 above)
Comment (mosaeedi) [reply src/pages/index.tsx:127]: I've uploaded the updated version with the correct copy!
I'll review it again.

File: src/pages/index.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ export default function Home() {
         <Star className="absolute right-80    top-40" />
```
Comment (kristoferlund) [inline src/pages/index.tsx:38]: New logo is missing in header. Design assets were posted by Zepti in the Discord group.

**Review by kristoferlund — APPROVED** [review]

File: src/pages/index.tsx (hunk 2 above)
Comment (kristoferlund) [reply src/pages/index.tsx:38]: Creating new issue instead #120

File: src/pages/index.tsx (hunk 1 above)
Comment (kristoferlund) [reply src/pages/index.tsx:127]: Creted a separate issue for this #121

**Comment by kristoferlund** [comment]
@yussdev I created new issues for things I noticed. Changes look good! Merging.

**Comment by mosaeedi** [comment]
Sorry, I forgot about this one.

@yussdev, if you still have issues with the lock/unlocked icon, please use these ones.

[SVG Icons](https://github.com/GeneralMagicio/pw-frontend/files/13266807/Archive.2.zip)

**Comment by mosaeedi** [comment]
I've commented on this issue! But nothing's here...


So, could you clarify what you mean by saying you need the whole SVG?
I've used icons from the Ionicons, but for whatever reason, exporting the lock icon from Figma does not produce the correct icon, so I've shared the original icon from the Ionicons.

Regarding the numbers, I'll fix them on Figma, but it's more about doing the formatting on the front end. It doesn't matter how it looks on Figma.

**Comment by kristoferlund** [comment]
@mosaeedi See my comments on the other issue, let's stop discussion here for same image. We are not doing any formatting of text and numbers for this section. We are using the whole block from Figma as **one image**. This is not an interactive part of the website but a static part of the landing page, an image. Since exporting to png, this is the one we are using: https://staging.pairwise.vote/images/voted-projects.png

**Pull request merged**